docs/custom-flows/error-handling.mdx

```diff
@@ -1,5 +1,100 @@
 ---
-sanity_slug: /docs/authentication/error-handling
+title: Error handling
```

Comment:
```suggestion
title: Error Handling
```

Do we use title case?  Unsure of what rules we follow here; but it appears that the [currently live page](https://clerk.com/docs/custom-flows/error-handling) uses title case.

Reply:
we use sentence-case!